client/src/app/components/Sidebar/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default () => {
+  return (
+    <div className='flex flex-col py-4 items-center'>
+      <Link to='/newgame'>
+        <CirclePlusIcon className='mb-4 text-gray-300 hover:text-white w-8' />
+      </Link>
+      <Link to='/home'>
+        <ListIcon className='mb-4 text-gray-300 hover:text-white w-8' />
+      </Link>
+      <Link to='/account'>
+        <UserIcon className='text-gray-300 hover:text-white w-8' />
+      </Link>
+    </div>
+  );
+};
+
+const CirclePlusIcon = props => (
+  <div {...props}>
+    <svg
```

Comment:
Can we instead import the tabler code somewhere (how do you "vendor" in js:#) and then follow this example? https://github.com/tabler/tabler-icons#svg-sprite I wanna make sure we're supporting them and using their MIT license appropriately.

Reply:
Perhaps, this is an example? 🤷 
 https://github.com/tabler/tabler/commit/4100ea4d34b550d187f9e6ba39c6f976085173b4#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R20

Reply:
👍, though the command there is copying _all_ SVGs from `node_modules` into a folder. How do we want to deal with hundreds of icons we don't use? Or is it not a big deal to have them all locally? I also removed some props from the copied `svg` so we can apply custom styles - not sure if things will conflict if we follow the above. I will try it out.

Reply:
I also want to make sure we're respecting their license, but I'm not sure how to proceed. Reading through the [MIT license](https://github.com/tabler/tabler-icons/blob/master/LICENSE), perhaps we can create one jsx file which exports any of their icons that we use, with the license copied at the top? This would seem to fulfill the license requirement:

```
Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
```

I tried playing around with the svg sprite example but could never get the icons to render properly - we just get an empty/transparent box (which still functions as a link) where the icon should be. I found [this](https://stackoverflow.com/questions/26815738/svg-use-tag-and-reactjs/26822815#32994814) and tried following it but no luck. [Here](https://github.com/tabler/tabler-icons#inline-html), the tabler-icon folks also mention simply inlining the `svg` tag itself... and [here](https://css-tricks.com/creating-svg-icon-system-react/) is a good article about doing so specifically in React.